adapted-articles/K/KingdomofGodKingdomofHeaven.xml

```diff
@@ -2,62 +2,62 @@
+<p class="fl">Jesus’s teachings about the kingdom of God embodied the same contrast between the present order and the future age as that of the prophets, and he expressed it in the idiom “this age and the age to come.” This fact is obscured in translations which use the word “world” instead of “age.” These are, however, two different concepts. A rich man asked Jesus what he must do to inherit eternal life (<a href="?bref=Mark.10.17">Mk 10:17</a>). The context makes it clear that he was asking about end-time life—the life of the resurrection (<a href="?bref=Dan.12.2">Dn 12:2</a>). Jesus speaks of the difficulty of entering the kingdom of God. (The parallel passage in <a href="?bref=Matt.19.23-24">Matthew 19:23-24</a> has both “kingdom of God” and “kingdom of heaven,” proving that they are interchangeable terms.) In their reaction the disciples ask, “Then who can be saved?” Jesus’s answer contrasts the portion of his disciples “in this time” with the “age to come” (<a href="?bref=Mark.10.29-30">Mk 10:29-30</a>) when they would inherit eternal life. It is clear from this passage that in some sense the kingdom of God, the kingdom of heaven, salvation, and eternal life all belong to the age to come. So far as this saying is concerned, God’s people will not experience eternal life until they do so in the new age.</p>
+<p>Matthew alone records the expression “the close of the age.” This age will be ended by the coming of the Son of Man (<a href="?bref=Matt.24.3">Mt 24:3</a>) and by the judgment of humanity (<a href="?bref=Matt.13.39-42">13:39-42</a>). Then the righteous will be separated from the wicked (v <a href="?bref=Matt.13.49">49</a>). The same expression occurs in the promise of the risen Jesus assuring his disciples of his presence until the end of the age (<a href="?bref=Matt.28.20">28:20</a>). It follows that if this age is to come to its end, it must be followed by another age—the age to come.</p>
+<p>The end-time kingdom will be inaugurated by an appearing—the glorious coming of the Son of Man. This is made clear by two of the parables about the kingdom of God. In the parable of the tares, “the Son of man will send his angels, and they will gather out of his kingdom all causes of sin and all evildoers, and throw them into the furnace of fire” (<a href="?bref=Matt.13.41-42">Mt 13:41-42</a>). The parable of the sheep and goats reflects the same end-time idea. When the Son of Man comes in his glory, he will sit on his glorious throne to judge the nations, separating the sheep from the goats. The righteous—the sheep—are to “inherit the kingdom prepared for you from the foundation of the world”; and entrance into the kingdom is synonymous with entrance into life (<a href="?bref=Matt.25.31-46">25:31-46</a>).</p>
```

Comment:
Christians tend to do runaway participles but this is too much. "by an appearance"